kong/plugins/response-transformer/header_transformer.lua

```diff
@@ -96,6 +96,10 @@ function _M.transform_headers(conf, headers)
   for _, old_name, new_name in iter(conf.rename.headers) do
     if headers[old_name] ~= nil and new_name then
       local value = headers[old_name]
+      -- header value can be a table, in which case we take the first value
+      if type(value) == "table" then
+        value = value[1]
+      end
       set_header(new_name, value)
```

Comment:
Since having multiple headers with the same name is a common practice, is it possible to keep both instead of only taking the first value?
Like this https://github.com/Kong/kong-ee/blob/f9c958ea2054f1c2d1745dc4f1ca2680c5d4fc6d/kong/pdk/response.lua#L535

Reply:
done